linux/build-environment/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM ubuntu:focal
+
+LABEL maintainer="d.vedenko@audacityteam.org"
+LABEL description="A build environment to check the builds for the package maintainers"
+LABEL version="3.0"
+
+ENV TZ=Europe/London
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
I personally think it is neater to put all `RUN` commands in a separate Bash script that gets copied into the Docker image. See this [example](https://github.com/musescore/MuseScore/blob/b30a63ff3822869cf9186bd92e1b9dc6983238a9/build/Linux%2BBSD/portable/x86_64/Dockerfile#L4-L5) for MuseScore, and also note the surrounding comments about how to reduce the image size (and how not to do it). [Here](https://github.com/musescore/MuseScore/blob/b30a63ff3822869cf9186bd92e1b9dc6983238a9/build/Linux%2BBSD/portable/Recipe#L116-L119) is where the tidying up happens.

Reply:
Nice catch, I will look into on Monday